apis/python/tests/test_dataframe.py

```diff
@@ -991,3 +991,10 @@ def test_timestamped_ops(tmp_path, allows_duplicates, consolidate):
         assert list(x.as_py() for x in tab["string"]) == ["apple"]
         assert sidf.tiledb_timestamp_ms == 1615402887987
         assert sidf.tiledb_timestamp.isoformat() == "2021-03-10T19:01:27.987000+00:00"
+
+
+def test_exists():
+    # Specifically, we're checking this doesn't raise
+    assert not soma.DataFrame.exists("/nonesuch/no/nope/never/ever")
+    assert not soma.DataFrame.exists("foo://bar")
+    assert not soma.DataFrame.exists("s3://@@@@ILLEGAL@@@@")
```

Comment:
@atolopko-czi I'll make `tests/test_tiledboject.py` and add negative cases for all the foundational and composite types.

I won't add positive cases since those already have _very_ thorough coverage in existing tests already, and I'll add a comment to that effect.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/1040/commits/1f9430cc56ae7d2ee00f2dfa130eb30dd21fecba